specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/azurelargeinstance.json

```diff
@@ -0,0 +1,1106 @@
+{
```

Comment:
Please apply for suppression (see suppression process in PR summary) for Linter error, to not implement PUT API.